apps/nova/src/components/Chat/Messages.tsx

```diff
@@ -270,9 +273,13 @@ const Messages: React.FC<{
               </ul>
             )}
           </div>
-
-          <div className="px-4 pt-2 pb-6">
-            <SendInput onSubmit={submit} />
+          {isTyping ? (
+            <div className="px-6">
+              <UserTyping {...otherMember} imageUrl={otherMember.image} />
```

Comment:
The other member has more values than what `UserTyping` needs. Let's provide explicit values.